PackageExplorer/Utilities/NuGetExtensions.cs

```diff
@@ -0,0 +1,271 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.Frameworks
+{
+	/// <summary>
+	/// Class for Extensions related to NuGetFramework objects.
+	/// </summary>
+	public static class NuGetFrameworkExtensions
+	{
+		#region Portable Framework Mappings
+		/// <summary>
+		/// Finds the short name of the by identifier.
+		/// </summary>
+		/// <param name="frameworkMapping">The framework mapping.</param>
+		/// <param name="frameworkIdentifier">The framework identifier.</param>
+		/// <param name="includeEquivalentFrameworks">if set to <c>true</c> [include equivalent frameworks].</param>
+		/// <returns>A collection of NuGetFramework items that match the framework specified.</returns>
+		public static IEnumerable<NuGetFramework> FindByIdentifierShortName(this IPortableFrameworkMappings frameworkMapping,
+			string frameworkIdentifier, bool includeEquivalentFrameworks = false)
+		{
+			var frameworkName =
+				DefaultFrameworkMappings.Instance.IdentifierShortNames.FirstOrDefault(
+					x =>
+						string.Compare(x.Value, frameworkIdentifier, StringComparison.InvariantCultureIgnoreCase) == 0 ||
```

Comment:
They are not the same and have the potential for null value exceptions.  Adding a null check would be just as complicated --- unless you want me to switch to using the ?. in C# 6?